Danyal_Khan/surveyform/apps/survey_form_app/templates/results.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>results</title>
+	</head>
+	<body>
+	<h3>Thanks for submitting this form. You have submitted this form  {{request.session.counter}} times</h3>
+		<h1>Submitted Information</h1>
+		<h3>Name: {{request.session.data.name}} </h3>
+		<h3>Dojo: {{request.session.data.dojo}}</h3>
+		<h3>Favorite language: {{request.session.data.language}}</h3>
+		<h3>Comment:</h3> <h4>{{request.session.data.comments}}</h4>
+	</body>
```

Comment:
You might want to add a 'go back' link somewhere on this page 
<a href="/">Go Back</a>